src/file_operations.js

```diff
@@ -21,6 +26,83 @@ const directoryExists = directory => {
   )
 }
 
+const downloadCourses = (coursesJson, coursesDir) => {
```

Comment:
I think maybe these two S3 related functions should go in a separate file - perhaps `src/course_download.js` ? it just seems like because they accomplish a fairly specific and separate thing from the rest of the functions in this file it would be better for them to live separately